content/docs/concepts/traffic-management/routing/service-entries/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Service Entries
+description: Learn about using a service entry to add an entry to Istio's abstract model that configures routing rules for external dependencies of the mesh.
+weight: 4
+keywords: [traffic-management, http, tcp, grpc, ingress, egress]
+---
+
+A [service entry](/docs/reference/config/networking/v1alpha3/service-entry)
+is a network resource used to add an entry to Istio's abstract model, or
+service registry, that Istio maintains internally. Once added, the Envoy
+proxies can send traffic to the external service as if it was a service in your
+mesh. You can configure service entries to configure routing rules to:
+
+-  Redirect and forward traffic for external destinations, such as APIs
+   consumed from the web, or traffic to services in legacy infrastructure.
+
+-  Define
+   [retry](/docs/concepts/traffic-management/network/#timeouts-and-retries),
+   [timeout](/docs/concepts/traffic-management/network/#timeouts-and-retries),
+   and [fault injection](/docs/concepts/traffic-management/network/#fault-injection)
+   policies for external destinations.
+
+- Add a service running in a Virtual Machine to the mesh to [expand your mesh](/docs/setup/kubernetes/additional-setup/mesh-expansion/#running-services-on-a-mesh-expansion-machine).
+
+- Logically add services from a different cluster to the mesh to configure a
+  [multicluster Istio mesh](/docs/tasks/multicluster/gateways/#configure-the-example-services)
+  on Kubernetes.
+
+You don’t need to add a service entry for every mesh-external service that you
+want your mesh services to use. By default, Istio configures the Envoy proxies
+to passthrough requests from unknown services. You can also use service entries
+to configure internal infrastructure:
+
+-  A **mesh-internal** service entry adds a service running in unmanaged
+   infrastructure, which does not have an Envoy proxy, but still is within the
```

Comment:
This doesn't sound right.

mesh-external is used for services without an Envoy proxy, regardless of their network.

mesh-internal is for services that are in the mesh, and therefore must have an Envoy proxy. The difference is that mesh-internal service entries are used to manually register mesh services that don't have a service discovery adapter that adds them to the registry automatically.

Reply:
I'll rewrite based on your feedback and push the new version.